Gems/Atom/Tools/ShaderManagementConsole/Scripts/GenerateShaderVariantListUtil.py

```diff
@@ -196,6 +208,11 @@ def create_shadervariantlist_for_shader(filename):
 
             valueName = shaderOptionDescriptor.GetValueName(optionValue)
             options[optionName] = valueName
+            if optionName.ToString() == "o_wrinkleLayers_count":
+                #print(optionValue)
+                print("----Value")
+                print(valueName.ToString())
+                print("----Value")
```

Comment:
Did you mean to remove this before committing?

Reply:
yes, apology